SPECS/mariner-rpm-macros/macros

```diff
@@ -93,6 +94,7 @@
     %{__os_install_post_leave_signatures} \
     %{?__brp_strip} \
     %{?mariner_ccache_enabled:CCACHE_DIR="/ccache-dir" ; export CCACHE_DIR ;} \
+    %{?mariner_ccache_enabled:CCACHE_COMPILERCHECK=content ; export CCACHE_COMPILERCHECK ;}\
```

Comment:
I'm not sure this change is needed. All we're doing here is printing the ccache stats. In fact I've considered removing the call to `ccache --show-stats` as it can clutter the logs and the numbers printed are cumulative, so perhaps not much value to the current build. While we're making a change to this file, i think we should either: (1) remove this call or (2) also call `ccache -z` to clear the stats at the beginning. What do you think?

Reply:
Makes sense, I will remove CCACHE_COMPILERCHECK from here.
For printing the stats, I think it's really helpful for debugging. The numbers in the official build are easy to read since it's always the first run on the cache and there's no aggregation. I see the problem for dev builds which are harder to read due to their cumulative ... Let me switch to -z...

Reply:
Sounds good, calling the `ccache -z` in the `__spec_build_pre` should fix this then